cabi/Cargo.toml

```diff
@@ -30,3 +30,4 @@ serde_json = "1.0.40"
 failure = "0.1.5"
 apple-crash-report-parser = { version = "0.4.0", features = ["with_serde"] }
 symbolic = { version = "7.3.3", path = "..", features = ["debuginfo", "demangle", "minidump", "proguard", "sourcemap", "symcache", "unreal-serde"] }
```

Comment:
Can the old C-ABI be shimmed with the new proguard version? It would be extremely nice to get rid of `symbolic-proguard` completely and deprecate the entire crate.